devicetypes/smartthings/zwave-temp-humidity-sensor.src/zwave-temp-humidity-sensor.groovy

```diff
@@ -0,0 +1,213 @@
+/**
+ *  Copyright 2020 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Generic Z-Wave Water/Temp/Humidity Sensor
+ *
+ *  Author: SmartThings
+ *  Date: 2020-07-22
+ */
+
+metadata {
+	definition(name: "Z-Wave Temp/Humidity Sensor", namespace: "smartthings", author: "SmartThings", mnmn: "SmartThings", "vid": "generic-humidity", ocfDeviceType: "oic.d.thermostat") {
```

Comment:
@ZWozniakS Since all the devices have another VID in their fingeprints - is it needed to define this VID in the DTH definition ?

Reply:
i thought that it would be useful to add default metadata as this DTH is made as a generic one